CONTRIBUTING.md

```diff
@@ -75,3 +75,15 @@ Use the following query string to check the comparator output:
 ```
 ?repo=testing-library%2Fdom-testing-library&from=v6.16.0&to=v8.1.0
 ```
+
+## Working on dark mode locally
```

Comment:
I added some docs on how to work with the dark mode locally. 😄 

If you have anything important you want me to know, please put it in here, so I can refer to this document while I'm colorzing things... 😄

Reply:
Nice addition! It won't be necessary eventually when we release the dark mode officially, but at that time we can convert this section into "How to enable env vars locally" I guess 😅